docs/content/components/layout/columns/columns.mdx

```diff
@@ -23,9 +24,9 @@ import { Columns } from '@marigold/components';
     {
       property: 'columns (required)',
       description:
-        'An array of numbers to set the size of the children. The columns array length and the count of children must be the same.',
-      type: 'number[]',
-      default: '',
+        'An array of numbers to set the size of the children. The columns array length and the count of children must be the same. Write "fit" for the column you want to have it fitting the contens width ',
+      type: 'number[] | "fit"',
```

Comment:
I think this type definition is not entirely correct. It should be **type: '(number | 'fit')[]** or can I write columns='fit?

Reply:
your right :)